Review Assignment/src/assets/scripts/about.js

```diff
@@ -0,0 +1,166 @@
+window.about = function() {
+  var aboutPage = {};
+  var type = 'GET';
+  var hideElement = 'hidden';
+  var click = 'click';
+
+  function setData(key, value) {
+    localStorage.setItem(key, value);
+  }
+
+  function getData(key) {
+    var value = localStorage.getItem(key);
+    return value;
+  }
+
+  function storeData(fullName, gender, birthDate, mStatus, location, occupation, skills, job) {
+    var newData =  {
+      'name': fullName,
+      'gender': gender,
+      'birthDate': birthDate,
+      'status': mStatus,
+      'location': location,
+      'occupation': occupation,
+      'skill': skills,
+      'job': job
+    };
+
+    localStorage.setItem('userData', JSON.stringify(newData));
+  }
+
+  function personData() {
+    const userData = JSON.parse(localStorage.getItem('userData'));
+    return userData;
+  }
+
+  function defaultUserInfo() {
+    var userData = personData();
+    if (userData == null) {
+      let dataUrl = '../assets/data/about.json';
+
+      $.get({url: dataUrl, type: type, success: function(userDetails)  {
+        storeData(userDetails.name, userDetails.gender, userDetails.birthDate, userDetails.status, userDetails.location, userDetails.occupation, userDetails.skills, userDetails.job);
+        updatePersonalInfo();
+      }});
+    }
+  }
+
+  //Default values of about info and work form deatails
+  function personalInfo() {
+    var userInfo = personData();
+    let userName = userInfo.name;
```

Comment:
Please make it consistent. Somewhere is `let` and somewhere is `var`. Preferring `let`.